packages/@lwc/ssr-compiler/src/__tests__/fixtures.spec.ts

```diff
@@ -84,10 +88,12 @@ describe.runIf(process.env.TEST_SSR_COMPILER).concurrent('fixtures', () => {
         {
             root: path.resolve(__dirname, '../../../engine-server/src/__tests__/fixtures'),
             pattern: '**/index.js',
+            expectedFailures,
         },
         async ({ filename, dirname, config }) => {
-            const errorFile = config?.ssrFiles?.error ?? 'error.txt';
-            const expectedFile = config?.ssrFiles?.expected ?? 'expected.html';
```

Comment:
I'm not a big fan of the `expected-ssr` system but I suppose we can debate bringing it back, since it is better to assert what _kind_ of error we expect to get. Maybe we can bring it back after all the main test failures are un-ignored.

Reply:
There are cases where `ssr-compiler` has an explicit divergence from `engine-server`, like explicitly not supporting deprecated `lwc:dynamic`. That's the intended use case for separate files.

Reply:
Yeah I get it. I can bring it back if you prefer.

Reply:
I think in this case it's better to exclude these few cases explicitly and assert the behavior in `ssr-compiler` rather than mixing the fixtures.

Reply:
04e7017f6

Reply:
I'm agnostic to how to handle these cases. Arguably these tiny divergences between engine-server and ssr-compiler will continue in perpetuity because we don't care to fix them (e.g. deprecated `lwc:dynamic`). So I'm happy to keep it in for now and defer the conversation.